test/quantization/core/test_quantized_op.py

```diff
@@ -4780,7 +4886,7 @@ def test_qconv2d(
            Y_scale=st.floats(4.2, 5.6),
            Y_zero_point=st.sampled_from([0]),
            use_bias=st.booleans(),
-           use_relu=st.booleans(),
+           post_op=st.sampled_from(["none", "relu"]),
```

Comment:
same here

Reply:
Thanks for the suggestions, split them into separate tests.